configure

```diff
@@ -220,6 +221,8 @@ usage()
     echo "    Compile with fuzzing"
     usage_with_default "--enable/disable-rust" "$RUST" "enable" "disable"
     echo "    Compile with Rust support"
+    usage_with_default "--enable/disable-grpc" "$GRPC" "enable" "disable"
+    echo "    Compile with cln-grpc plugin"
```

Comment:
If this is enabled but RUST isn't, iiuc it won't actually compile the CLN-GRPC plugin right? Seems confusing.